tests/testthat/test-h_bracket.R

```diff
@@ -0,0 +1,27 @@
+
+
+
+test_that("h_bracket works as expected", {
+    st_mod <- StanModule(
+        functions = "string",
+        data = "string",
+        parameters = "string",
+        transformed_parameters = "string",
+        generated_quantities = "string",
+        priors = list("prior1" = "item1;","prior2" = "item2;")
+    )
+
+    expect_equal(h_bracket(st_mod@functions), "{\nstring\n}\n")
+    expect_equal(h_bracket(st_mod@data), "{\nstring\n}\n")
+    expect_equal(h_bracket(st_mod@parameters), "{\nstring\n}\n")
+    expect_equal(h_bracket(st_mod@transformed_parameters), "{\nstring\n}\n")
+    expect_equal(h_bracket(st_mod@generated_quantities), "{\nstring\n}\n")
+})
+
+
+test_that("Empty strings work fine", {
+    str <- ""
+    actual <- h_bracket(str)
+    expected <- NULL
+    expect_equal(actual, expected)
+})
```

Comment:
this is the unit test to ensure that `h_bracket` works fine when the string is empty. Note that the result is different if the user provides a string with one empty space `" "`. I don't see why this will ever happen but if you consider it important I can add some code to avoid this case.

Reply:
Might come back to this later but I think its fine for now to get this closed